streams/examples/src/main/java/org/apache/kafka/streams/examples/pageview/PageViewTypedJob.java

```diff
@@ -124,9 +112,9 @@ public static void main(String[] args) throws Exception {
         serdeProps.put("JsonPOJOClass", RegionCount.class);
         regionCountSerializer.configure(serdeProps, false);
 
-        KStream<String, PageView> views = builder.stream(stringDeserializer, pageViewDeserializer, "streams-pageview-input");
+        KStream<String, PageView> views = builder.stream(Serdes.String(), Serdes.serdeFrom(null, pageViewDeserializer), "streams-pageview-input");
```

Comment:
> Are there really situations where users only need one of serializer/deserializer outside of toy examples?

Yeah.  It might be a good idea to force users to always provide a fully implement serde, i.e. with both serializer and deserializer functionality.

WDYT, @guozhangwang ?